models/baseModels/Invoice/Invoice.ts

```diff
@@ -166,6 +166,9 @@ export abstract class Invoice extends Transactional {
     await super.afterSubmit();
 
     // update outstanding amounts
+    if (this.schemaName === 'SalesQuote') {
+      return;
+    }
```

Comment:
Place these lines before the comment to not be confused as its part of the below code, outstanding amount